tests/components/conftest.py

```diff
@@ -804,12 +810,26 @@ async def _check_create_issue_translations(
         )
 
 
+def _get_request_quality_scale(
+    request: pytest.FixtureRequest, rule: str
+) -> Literal["done", "exempt", "todo"]:
```

Comment:
Can we create a `StrEnum` from `Literal["done", "exempt", "todo"]` as than we have only one place to update if in the future we add another value